### PR TITLE
I18n sync HOC 11/27

### PR DESCRIPTION
**Pull request by Hamms**

### CODE REVIEW

File: i18n/locales/en-GB/hourofcode/how-to/companies.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-title: 怎样把《编程一小时》带到你的公司
+title: '怎样把《编程一小时》带到你的公司'
```
Comment (joshlory) [inline i18n/locales/en-GB/hourofcode/how-to/companies.md:2]: This doesn't look right?

File: pegasus/sites.v3/hourofcode.com/i18n/public/gb/how-to/companies.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ---
-title: 怎样把《编程一小时》带到你的公司
+title: '怎样把《编程一小时》带到你的公司'
```
Comment (joshlory) [inline pegasus/sites.v3/hourofcode.com/i18n/public/gb/how-to/companies.md:2]: Also here ^

File: i18n/locales/en-GB/hourofcode/how-to/companies.md (hunk 1 above)
Comment (joshlory) [reply i18n/locales/en-GB/hourofcode/how-to/companies.md:2]: https://hourofcode.com/us/gb/how-to/companies

File: i18n/locales/fr-FR/hourofcode/thanks.md [hunk 3]
```diff
@@ -3,18 +3,18 @@ title: <%=hoc_s(titre_inscription_merci) %>
 layout: wide
 nav: how_to_nav
 social:
-  "og:title": '<%=hoc_s(:tag_meta_titre_og) %>'
+  "og:title": <%=hoc_s(:tag_meta_titre_og) %>
   "og:description": <%= hoc_s(tag_meta_déscription_og) %>
-  "og:image": 'http://<%=requête.hôte%>/images/heureducode-2015-vignette-vidéo.png'
+  "og:image": http://<%=requête.hôte%>/images/heureducode-2015-vignette-vidéo.png
```
Comment (joshlory) [inline i18n/locales/fr-FR/hourofcode/thanks.md:8]: Translated method call...

File: i18n/locales/fr-FR/hourofcode/thanks.md [hunk 4]
```diff
@@ -3,18 +3,18 @@ title: <%=hoc_s(titre_inscription_merci) %>
...
   "twitter:card": joueur
   "twitter:site": '@code. org'
-  "twitter:url": 'http://<%=requête.hôte%>'
-  "twitter:title": '<%= hoc_s(:étiquette_meta_twitter_titre) %>'
-  "twitter:description": '<%=hoc_s(:étiquette_meta_twitter_déscription) %>'
-  "twitter:image:src": 'http://<%=requête.hôte%>/images/heureducode-2015-vignette-vidéo.png'
+  "twitter:url": http://<%=requête.hôte%>
+  "twitter:title": <%= hoc_s(:étiquette_meta_twitter_titre) %>
+  "twitter:description": <%=hoc_s(:étiquette_meta_twitter_déscription) %>
+  "twitter:image:src": http://<%=requête.hôte%>/images/heureducode-2015-vignette-vidéo.png
```
Comment (joshlory) [inline i18n/locales/fr-FR/hourofcode/thanks.md:17]: Translated i18n keys...

Comment (joshlory) [reply i18n/locales/fr-FR/hourofcode/thanks.md:17]: Why don't I repro this on https://hourofcode.com/fr/thanks?

File: pegasus/sites.v3/hourofcode.com/i18n/it.yml [hunk 5]
```diff
@@ -1290,17 +1290,17 @@ it:
...
-  tutorial_foos_longdescription: "Programma gli amabile Foos per risolvere gli enigmi. Poi entra in Foo Studio per creare e condividere i tuoi giochi. Ognuno può imparare a programmare ovunque si trovi con l'interfaccia award-winning \"senza parole\" dei Foos.  Gioca ora!"
+  tutorial_foos_longdescription: "\\t\nCodifica gli adorabili Foos per risolvere vari puzzle che insegnano i concetti fondamentali d'informatica come sequenza e cicli. Chiunque da qualunque parte del mondo può imparare a scrivere codice, grazie alla premiata interfaccia \"niente parole\" della codeSpark Academy. Sono benvenuti tanto programmatori alle prime armi quanto chi ancora non sa leggere!"
```
Comment (joshlory) [inline pegasus/sites.v3/hourofcode.com/i18n/it.yml:1303]: This looks not good?

File: i18n/locales/fr-FR/hourofcode/thanks.md [hunk 6]
```diff
@@ -3,18 +3,18 @@ title: <%=hoc_s(titre_inscription_merci) %>
...
+  "og:title": <%=hoc_s(:tag_meta_titre_og) %>
   "og:description": <%= hoc_s(tag_meta_déscription_og) %>
-  "og:image": 'http://<%=requête.hôte%>/images/heureducode-2015-vignette-vidéo.png'
+  "og:image": http://<%=requête.hôte%>/images/heureducode-2015-vignette-vidéo.png
   "og:image:width": 1440
   "og:image:height": 900
-  "og:url": 'http://<%=requête.hôte%>'
+  "og:url": http://<%=requête.hôte%>
   "twitter:card": joueur
   "twitter:site": '@code. org'
```
Comment (joshlory) [inline i18n/locales/fr-FR/hourofcode/thanks.md:13]: I think these should be `player` and `code.org`.

File: i18n/locales/en-GB/hourofcode/how-to/companies.md (hunk 1 above)
Comment (Hamms) [reply i18n/locales/en-GB/hourofcode/how-to/companies.md:2]: this is fine; just YAML being inconsistent

Comment (Hamms) [reply i18n/locales/en-GB/hourofcode/how-to/companies.md:2]: Oh! Yeah, this is in English

**Pull request merged**